### ISSUE TITLE
Chrome: <slot> elements don't display

### ISSUE THREAD

**Issue by sk1p** [salesagility/SuiteCRM#3232]
#### Issue
Some pages (two examples: Inbound Email settings, Currency list) don't display some elements (forms elements, content fields) on Chrome with enabled Experimental Web Platform features (see chrome://flags/#enable-experimental-web-platform-features )

![Chrome](https://cloud.githubusercontent.com/assets/5778/23654869/fd856904-0332-11e7-9c7a-a38829345f41.png)

It works fine in Firefox (or Chrome with the flag disabled):

![Firefox](https://cloud.githubusercontent.com/assets/5778/23654887/128a0292-0333-11e7-9bbe-dfc727c363d6.png)

#### Expected Behavior
All form and content elements should display

#### Actual Behavior
Some elements are not displayed.

#### Possible Fix
A common theme among the broken pages is the usage of ``<slot>`` HTML tags. Assigning ``display: block`` to those works around the bug. A proper fix would be to remove ``<slot>`` usage completely (I guess?). According to [a Chomium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=660265), "no one [is] using <slot> in non-shadow trees in real world." and "there is no useful use case for a slot in a non-shadow tree in real world." Can someone explain the ``<slot>`` usage in SuiteCRM, and if they are superfluous, should I prepare a pull request for removing them?

#### Steps to Reproduce
1. Enable Experimental Web Platform features
2. Login as administrator
3. Navigate to ``/index.php?module=Currencies&action=index``
4. Note that the table columns that should contain the currencies don't display

#### Context
Because the behaviour hidden behind the flag might become standard at some point in the future, it is best to fix this ASAP

#### Your Environment
* SuiteCRM Version used: 7.8.2
* Browsers: works on Firefox ESR 45.7.0, broken Chrome 56.0.2924.87 (64-bit) running on Debian Unstable and Windows 7
* Switching themes does not fix the bug

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Chrome: <slot> elements don't display](https://trello.com/c/d2AKxvgJ/17-chrome-slot-elements-don-t-display)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
